index.html

```diff
@@ -671,16 +909,60 @@
 	let output = `Generated by,${author}, \nDate,${curDate}, \n
 	Pitch Deviations are from ET\n
 	Width Deviations are from Just\n\n
-	cPitchETDev,ePitchETDev,5M WidthJIDev,3M WidthJIDev,Comment, \n`;
+	row&col,cPitchETDev,ePitchETDev,5M WidthJIDev,3M WidthJIDev\n`;
+
+	// slotData.forEach( slot => {
+	// 	output += `${slot.cFromET}, ${slot.eFromET}, ${slot.fifthDeviation}, ${slot.thirdDeviation}, ${slot.comment}\n`;
+	// })
 
-	slotData.forEach( slot => {
-		output += `${slot.cFromET}, ${slot.eFromET}, ${slot.fifthDeviation}, ${slot.thirdDeviation}, ${slot.comment}\n`;
-	})
+	// changes : I removed the 'comment' column because I am using it as a way to specify columns. I added an 'id' column that contains
+	// information about rows and columns of slots. 
+	TuningMap.forEach(function(value, key, map){
+			output += `${key},${value.cFromET}, ${value.eFromET}, ${value.fifthDeviation}, ${value.thirdDeviation}\n`;
```

Comment:
The purpose of adding separate columns for different quality is to easily process the data. Here you've obfuscated it by hiding it behind `key` which is the row & column in the same string.

I suggest the output format to have something like this:

|data | data | data | quality|
|--|--|--|--|
| || | good|
| || | bad |
| || | medium|
|...|...|...|...|

Spreadsheet programs have the ability to generate graphs and plots with legend according to Quality column.